app/Config/routes.php

```diff
@@ -14,30 +14,31 @@
  * Redistributions of files must retain the above copyright notice.
  *
  * @copyright     Copyright (c) Cake Software Foundation, Inc. (http://cakefoundation.org)
+ *
  * @link          http://cakephp.org CakePHP(tm) Project
- * @package       app.Config
  * @since         CakePHP(tm) v 0.2.9
+ *
  * @license       http://www.opensource.org/licenses/mit-license.php MIT License
  */
-/**
- * Here, we are connecting '/' (base path) to controller called 'Pages',
- * its action called 'display', and we pass a param to select the view file
- * to use (in this case, /app/View/Pages/home.ctp)...
- */
-	Router::connect('/', array('controller' => 'pages', 'action' => 'display', 'home'));
-/**
+    /**
+     * Here, we are connecting '/' (base path) to controller called 'Pages',
+     * its action called 'display', and we pass a param to select the view file
+     * to use (in this case, /app/View/Pages/home.ctp)...
+     */
+    Router::connect('/', array('controller' => 'pages', 'action' => 'display', 'home'));
+/*
  * ...and connect the rest of 'Pages' controller's URLs.
  */
-	Router::connect('/pages/*', array('controller' => 'pages', 'action' => 'display'));
+    Router::connect('/pages/*', array('controller' => 'pages', 'action' => 'display'));
 
-/**
+/*
  * Load all plugin routes. See the CakePlugin documentation on
  * how to customize the loading of plugin routes.
  */
-	CakePlugin::routes();
+    CakePlugin::routes();
 
-/**
- * Load the CakePHP default routes. Only remove this if you do not want to use
- * the built-in default routes.
- */
-	require CAKE . 'Config' . DS . 'routes.php';
+    /**
```

Comment:
__[staging][code_sniffer]__
* Block comments must be started with /*
* Inline doc block comments are not allowed; use "/* Comment */" or "// Comment" instead